inc/saibuffer.h

```diff
@@ -397,6 +397,12 @@ typedef enum _sai_buffer_pool_stat_t
     /** Get/set WRED marked bytes count [uint64_t] */
     SAI_BUFFER_POOL_STAT_WRED_ECN_MARKED_BYTES = 0x00000012,
 
+    /** Get current headroom pool occupancy in bytes [uint64_t] */
+    SAI_BUFFER_POOL_STAT_XOFF_CURR_OCCUPANCY_BYTES = 0x00000013,
+
+    /** Get headroom pool occupancy in bytes [uint64_t] */
+    SAI_BUFFER_POOL_STAT_XOFF_WATERMARK_BYTES = 0x00000014,
+
```

Comment:
Calling it as xxx_XOFF_ROOM_xxx will be aligned with the other attributes where we denote the headroom (e.g, SAI_INGRESS_PRIORITY_GROUP_STAT_XOFF_ROOM_CURR_OCCUPANCY_BYTES). Just a suggestion.